go/cmd/vtctldclient/command/vreplication/migrate/migrate.go

```diff
@@ -0,0 +1,148 @@
+	}
+)
+
+var createOptions = struct {
+	MountName                    string
+	SourceKeyspace               string
+	AllTables                    bool
+	IncludeTables                []string
+	ExcludeTables                []string
+	SourceTimeZone               string
+	NoRoutingRules               bool
+	Cells                        []string
+	TabletTypes                  []topodatapb.TabletType
+	TabletTypesInPreferenceOrder bool
+	OnDDL                        string
+	DeferSecondaryKeys           bool
+	AutoStart                    bool
+	StopAfterCopy                bool
+}{}
+
+var createCommand = &cobra.Command{
+	Use:                   "create",
+	Short:                 "Create and optionally run a Migrate VReplication workflow.",
+	Example:               `vtctldclient --server localhost:15999 migrate --workflow import --target-keyspace customer create --source-keyspace commerce --mount-name ext1 --tablet-types replica`,
+	SilenceUsage:          true,
+	DisableFlagsInUseLine: true,
+	Aliases:               []string{"Create"},
+	Args:                  cobra.NoArgs,
+	PreRunE: func(cmd *cobra.Command, args []string) error {
+		// Either specific tables or the all tables flags are required.
+		if !cmd.Flags().Lookup("tables").Changed && !cmd.Flags().Lookup("all-tables").Changed {
+			return fmt.Errorf("tables or all-tables are required to specify which tables to move")
+		}
+		if err := common.ParseAndValidateCreateOptions(cmd); err != nil {
+			return err
+		}
+		return nil
+	},
+	RunE: commandCreate,
+}
+
+func commandCreate(cmd *cobra.Command, args []string) error {
+	tsp := common.GetTabletSelectionPreference(cmd)
+	cli.FinishedParsing(cmd)
+
+	req := &vtctldatapb.MigrateCreateRequest{
+		Workflow:                  common.BaseOptions.Workflow,
+		TargetKeyspace:            common.BaseOptions.TargetKeyspace,
+		SourceKeyspace:            createOptions.SourceKeyspace,
+		MountName:                 createOptions.MountName,
+		SourceTimeZone:            createOptions.SourceTimeZone,
+		Cells:                     createOptions.Cells,
+		TabletTypes:               createOptions.TabletTypes,
+		TabletSelectionPreference: tsp,
+		AllTables:                 createOptions.AllTables,
+		IncludeTables:             createOptions.IncludeTables,
+		ExcludeTables:             createOptions.ExcludeTables,
+		OnDdl:                     createOptions.OnDDL,
+		DeferSecondaryKeys:        createOptions.DeferSecondaryKeys,
+		AutoStart:                 createOptions.AutoStart,
+		StopAfterCopy:             createOptions.StopAfterCopy,
+		NoRoutingRules:            createOptions.NoRoutingRules,
+	}
+
+	_, err := common.GetClient().MigrateCreate(common.GetCommandCtx(), req)
+	if err != nil {
+		return err
+	}
+
+	return nil
+}
+
+func addCreateFlags(cmd *cobra.Command) {
+	cmd.PersistentFlags().StringVar(&createOptions.SourceKeyspace, "source-keyspace", "", "Keyspace where the tables are being moved from.")
+	cmd.MarkPersistentFlagRequired("source-keyspace")
```

Comment:
No need to use persistent here unless we plan to add child commands (that would inherit it)